### PR TITLE
Bugfix/retransmission block on timeout

### PR DESCRIPTION
**Pull request by cavus700**
Since we do not know if the timeout while waiting is due to a lost udp ack packet or if our sent packet is lost, we need to resend our packet after timeout

### CODE REVIEW

**Comment by rageagainsthepc** [comment]
TCP reimplementation much? :P

**Comment by baez90** [comment]
NSTFTP = not so trivial file transfer protocol? 😜

**Comment by cavus700** [comment]
We should do our own TFTP with Blackjack and TCP :D

File: pkg/tftp/rrq_response.go [hunk 1]
```diff
@@ -277,17 +277,17 @@ func (r *rrqResponse) sendError(code ErrorCode, occurredError error) (err error)
...
 
-	_ = r.connection.SetWriteDeadline(time.Now().Add(5 * time.Second))
-	numBytes, err = r.connection.Write(buffer)
-	if err != nil {
-		return err
-	}
-
-	r.logger.Debug("Package send",
-		zap.Int("Bytes", numBytes))
+		r.logger.Debug("Package send",
```
Comment (rageagainsthepc) [inline pkg/tftp/rrq_response.go:288]:
```suggestion
		r.logger.Debug("Package sent",
```

**Review by rageagainsthepc — APPROVED** [review]
LGTM apart from a small grammatical error

**Pull request merged**